api/MediaCapabilities.json

```diff
@@ -0,0 +1,207 @@
+{
+  "api": {
+    "MediaCapabilities": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaCapabilities",
+        "description": "MediaCapabilities Interface",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "66"
+          },
+          "chrome_android": {
+            "version_added": "66"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "63"
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "55"
+          },
+          "opera_android": {
+            "version_added": "55"
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
+          "deprecated": false
+        }
+      },
+      "encodingInfo": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/MediaCapabilities/encodingInfo",
```

Comment:
I restarted the Travis CI test and it looks like this (and several other URLs) need the `en-us` bit dropped